app/views/apps/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="content">
+
+  <h1 class="header">Applications for <%= @pet.name %> </h1>
+
+  <h2> Applicant Name: </h2>
+  <br>
+  <% @pet.apps.each do |app| %>
+  <section id = "app-<%= app.id %>"  >
+    <p><a class= "pet-link" href="/apps/<%= app.id %>"><%= app.name %></a></p>
```

Comment:
We are using both the <a> and the link_to tags. if we have time we should standardize. add it to the tech debt